airflow/config_templates/default_airflow.cfg

```diff
@@ -550,6 +550,7 @@ hide_sensitive_variable_fields = True
 elasticsearch_host =
 elasticsearch_log_id_template = {{dag_id}}-{{task_id}}-{{execution_date}}-{{try_number}}
 elasticsearch_end_of_log_mark = end_of_log
+elasticsearch_frontend =
```

Comment:
NIT: We can have an example comment here to make it easier to adapt.

Reply:
Done!